src/app/helpers.ts

```diff
@@ -1,12 +1,12 @@
 import BigNumber from 'bignumber.js';
 import QRCode from 'qrcode';
 
-export const convertSatsToBch = (sats: number) => {
-  return +new BigNumber(sats).div(100000000).toFormat(8);
+export const convertSatsToBch = (sats: BigNumber) => {
+  return sats.div(100_000_000);
```

Comment:
@EyeOfPython Wait is 100_100_100 a BigNumber feature? You can't do this normally right 😅.

Reply:
you can do that in TypeScript but not in JavaScript :P

Reply:
Good to know 👍